include/s3select.h

```diff
@@ -2147,34 +2140,32 @@ class csv_object : public base_s3object
 
       m_previous_line = true;//it means to skip last line
 
+      csv_string = csv_string.substr(0, csv_string.length() - m_last_line.length());
     }
 
-    return run_s3select_on_object(result, csv_stream, stream_length, m_skip_first_line, m_previous_line, (m_processed_bytes >= obj_size));
-
+    return run_s3select_on_object(result, csv_string.c_str(), csv_string.length(), m_skip_first_line, m_previous_line, (m_processed_bytes >= obj_size));
```

Comment:
In this section, identify a broken row and save it for later(for merge).
it marks to skip this broken line.